docs/events.md

```diff
@@ -17,6 +17,13 @@ functions:
         input:
           key: value
           key2: value2
+    - nats:
+        name: my-nats-event
```

Comment:
nit: I think it's possible to include this directly in the top-level message, since all triggers can be given a name:

```yaml
events:
 - name: my-nats-event
   nats:
     scw_nats_config:
       ...
     
```

However, it's okay like this, the API doesn't define a clear standard for what's a trigger and I don't reckon it's a big deal.